client/app/templates/template-editor.html

```diff
@@ -0,0 +1,73 @@
+<div class="ss-details-wrapper">
+  <div class="panel panel-default ss-details-panel designer-catalog-details">
+    <div class="panel-body">
+      <section>
+        <div class="col-md-12 ss-details-header">
+          <div class="row">
+            <div class="col-lg-7 col-md-6 col-sm-6 col-xs-6">
+              <h2 translate ng-if="vm.template.id">Edit Template</h2>
+              <h2 translate ng-if="!vm.template.id">Add Template</h2>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="ss-form">
```

Comment:
not seeing the form enclosed in a`<form>` element or given an name, critical for employing form feedback